int32 num_humans
GestureDetectorHuman[] gesture_list
